lib/rover.rb

```diff
@@ -0,0 +1,48 @@
+class Rover
+  attr_accessor :name, :x, :y, :direction, :move, :compass_no
+
+  def initialize(name,x,y,compass_no)
+    @name = name
+    @x = x
+    @y = y
+    @direction = ['N','E','S','W']
+    @compass_no = compass_no
```

Comment:
It's hard to tell that this is the rover's direction.  `@direction` would be clearer.